spec/game_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './classes/game'
+require 'date'
+
+describe Game do
+  before :each do
+    @game = Game.new '2018/06/15', true, '2023/07/04'
+  end
+
+  describe '#new' do
+    it 'takes three parameters and returns a Game object' do
+      @game.should be_an_instance_of Game
+    end
+  end
+
+  describe '#multiplayer' do
+    it 'returns the correct publish_date' do
+      @game.multiplayer.should be_truthy
+    end
+  end
+  describe '#multiplayer' do
+  it 'returns the correct publish_date' do
+    @game.last_played_at.should eql "2023/07/04"
+    end
+  end
+  describe '#publish_date' do
+  it 'returns the correct publish_date' do
+    @game.publish_date.should eql '2018/06/15'
```

Comment:
use of `should` is **deprecated**.
I suggest you use `expect` instead

For example:
sh
```
expect(@game).to be_an_instance_of(Game)
expect(@game.multiplayer).to be true
expect(@game.last_played).to be eq('your date')
expect(@game.publish_date).to be eq('your date)
```